### PR TITLE
[Search] Update REST commit to latest from main - b8d48bca

### PR DESCRIPTION
**Pull request by Mohit-Chakraborty**

### CODE REVIEW

**Comment by Mohit-Chakraborty** [comment]
cc: @tg-msft

File: sdk/search/Azure.Search.Documents/src/autorest.md [hunk 1]
```diff
@@ -1,20 +1,14 @@
 # Azure.Search.Documents Code Generation
 
-Run `/sdk/search/generate.ps1` to generate code.
```
Comment (tg-msft) [inline sdk/search/Azure.Search.Documents/src/autorest.md:3]: Nit - can we replace with the dotnet command required or a link to where folks can find the latest from Contributing.md?

File: sdk/search/Azure.Search.Documents/src/Generated/Models/QueryLanguage.cs [hunk 2]
```diff
@@ -161,9 +161,9 @@ public QueryLanguage(string value)
         public static QueryLanguage ArJo { get; } = new QueryLanguage(ArJoValue);
         /// <summary> Query language value for Danish (Denmark). </summary>
         public static QueryLanguage DaDk { get; } = new QueryLanguage(DaDkValue);
-        /// <summary> Query language value for Norwegian (Normway). </summary>
```
Comment (tg-msft) [inline sdk/search/Azure.Search.Documents/src/Generated/Models/QueryLanguage.cs:164]: I'm surprised the swagger linter didn't complain...?

**Review by tg-msft — APPROVED** [review]

File: sdk/search/Azure.Search.Documents/src/Generated/Models/QueryLanguage.cs (hunk 2 above)
Comment (Mohit-Chakraborty) [reply sdk/search/Azure.Search.Documents/src/Generated/Models/QueryLanguage.cs:164]: I found a company with this name. Internet does not tell me what the word means.

**Pull request merged**